src/components/Slideover/Slideover.tsx

```diff
@@ -1,13 +1,30 @@
+import { useCallback, useEffect } from "react"
 import styles from "./Slideover.module.scss"
 
 export default function Slideover(props: { status: boolean; changeStatus: any; wojakId: number }) {
   const { status, changeStatus, wojakId } = props
 
+  const escListener = useCallback(
+    e => {
+      if (e.key === "Escape") {
+        changeStatus(false)
+      }
+    },
+    [changeStatus],
+  )
+
+  useEffect(() => {
+    window.addEventListener("keyup", escListener)
+    return () => {
+      window.removeEventListener("keyup", escListener)
+    }
+  }, [escListener])
+
   return (
     <>
       {status ? (
         <section className={styles.slideoverSection} role="dialog" aria-modal="true">
-          <div className={styles.bgOverlayContainer}>
+          <div className={styles.bgOverlayContainer} onClick={() => changeStatus(false)}>
```

Comment:
I think this okay but going this way, the user will not be able to click on any details in the slideover.
Is this a desired behavior?

Reply:
Good catch, wanted to do it on overlay, not container, thanks!